drivers/backup/auth.go

```diff
@@ -849,6 +849,33 @@ func GetAllGroups() ([]KeycloakGroupRepresentation, error) {
 	return groups, nil
 }
 
+func GetAllUsers() ([]KeycloakUserRepresentation, error) {
```

Comment:
Why are we returning `KeycloakUserRepresentation`? I think in most cases we just need a slice of usernames.

Reply:
we are using **GetAllUsers()** helper function in other place as well , that's why didn't make changes to it